app/views/application/_usb.html

```diff
@@ -1,14 +1,12 @@
-<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="" height="75" viewBox="0 0 73 75">
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="58" height="50" viewBox="0 0 58 50">
```

Comment:
This size change will also impact the U2F authentication flow. We should confirm that looks correct before landing.

Reply:
I couldn't find anywhere else where this partial was being used, but if we need to we can resize via CSS in one place or another.

Reply:
Ah, I see it here:
https://github.com/brave-intl/publishers/blob/aa6eab1fcc7af350499f71d0878977e1f6693e06/app/views/two_factor_authentications/_u2f.html.slim#L25

Reply:
@mixonic I looked into this, and it seems my "new" version is actually the exact same rendered size for the icon itself, but without added padding and rotation information included in the SVG file. This seems to be an improvement in the export itself, and it still looks good when shown during auth:
![USB icon during 2FA auth](https://user-images.githubusercontent.com/250934/34592589-464c426c-f193-11e7-87f6-2b79b9859c26.png)